.github/workflows/integrationTest.yml

```diff
@@ -610,7 +610,7 @@ jobs:
             fi
       
       - name: Terraform destroy
-        if: ${{ cancelled() && steps.ecs-fargate-integration-test.outputs.cache-hit != 'true'
+        if: ${{ cancelled() && steps.ecs-fargate-integration-test.outputs.cache-hit != 'true' }}
```

Comment:
I was going to put in a PR for this today after I saw that my fork failed because of it. Would rather get that fix in sooner than including it in this PR.

Reply:
Andrew has created a separated PR for it https://github.com/aws/amazon-cloudwatch-agent/pull/461.

Reply:
It will auto resolve then if this fix is already in.